calico-enterprise/multicluster/create-a-management-cluster/create-a-management-cluster-helm.mdx

```diff
@@ -66,37 +69,129 @@ If you are installing on a cluster installed by EKS, GKE, AKS or Mirantis Kubern
 
 1. Add any other customizations you require to `values.yaml`. To see values that can be customized in the chart, see the [helm docs](https://helm.sh/docs/) or run the following command:
 
-  ```
-  helm show values projectcalico/tigera-operator --version tigera-operator-{{version}}
-  ```
+  <CodeBlock language='bash'>
+     {'{{version}}' === 'master'
+       ? `helm show values tigera/tigera-operator --version v0.0`
+       : `helm show values ./tigera-operator-{{chart_version_name}}.tgz`}
+  </CodeBlock>
+
 
 ### Install {{prodname}}
 
+<Tabs>
+<TabItem label="NodePort" value="NodePort">
+
 To install a {{prodname}} [management](../create-a-management-cluster#value) cluster with Helm:
 
 1. [Configure a storage class for Calico Enterprise](../../operations/logstorage/create-storage).
 
-1. Export the service port number, and the public IP or host of the management cluster. (Ex. "example.com:1234" or "10.0.0.10:1234".)
+1. Export the service node port number
 
   ```bash
-  export MANAGEMENT_CLUSTER_ADDR=<your-management-cluster-addr>
+  export EXT_SERVICE_NODE_PORT=30449
+  ```
+
+  Export the public address or host of the management cluster. (Ex. "example.com:1234" or "10.0.0.10:1234".)
+
+  ```bash
+  export MANAGEMENT_CLUSTER_ADDR=<your-management-cluster-addr>:$EXT_SERVICE_NODE_PORT
   ```
 
-1. Export the service type (`NodePort` or `LoadBalancer`), the service port, target port and protocol.
+1. Export the service type, port, target port and protocol.
   
   ```bash
   export EXT_SERVICE_TYPE=NodePort
   export EXT_SERVICE_PORT=9449
   export EXT_SERVICE_TARGET_PORT=9449
   export EXT_SERVICE_PROTOCOL=tcp
```

Comment:
To one of your original questions regarding defaults - I'm thinking maybe these could be defaults in the values.yaml